examples/query.ts

```diff
@@ -10,16 +11,18 @@ const queryApi = new InfluxDB({url, token}).getQueryApi(org)
 const fluxQuery =
   'from(bucket:"my-bucket") |> range(start: -1d) |> filter(fn: (r) => r._measurement == "temperature")'
 
-console.log('*** QUERY ROWS ***')
 // There are more ways of how to receive results,
-// the essential ones are shown/commented below. See also rxjs-query.ts .
-//
-// Execute query and receive table metadata and rows as they arrive from the server.
-// https://docs.influxdata.com/influxdb/latest/reference/syntax/annotated-csv/
-queryApi.queryRows(fluxQuery, {
-  next: (row: string[], tableMeta: FluxTableMetaData) => {
+// the essential ones are shown in functions below.
+// Execution of a particular function follows
+// its defintion, comment/uncomment it at will.
+// See also rxjs-query.ts and queryWithParamas.mjs .
```

Comment:
I don't see `rxjs-query.ts` file

Reply:
it is in the same directory

Reply:
... but there was a type in `queryWithParams.mjs` example
```bash
influxdb-client-js/examples$ ls rxjs-query.ts queryWithParams.mjs
queryWithParams.mjs  rxjs-query.ts
```

Reply:
> it is in the same directory

Ok, I thought `rxjs-query.ts` is a new file.